rules/falco_rules.yaml

```diff
@@ -2417,6 +2421,8 @@
     - name: proc_name_image_suffix
       fields: [proc.name, container.image.repository]
       comps: [in, endswith]
+      values:
+        - [[rpm], quay.io/sysdig/node-image-analyzer]
```

Comment:
@mstemm , I think `node-image-analyzer` will go to the customization on our side, not in OSS

Reply:
Agree